src/services/transcript-parser.js

```diff
@@ -12,11 +12,21 @@ import {
   parseSequences,
 } from './utility-helpers';
 
-const TRANSCRIPT_MIME_TYPES = [
-  { type: 'application/json', ext: 'json' },
-  { type: 'text/vtt', ext: 'vtt' },
-  { type: 'text/plain', ext: 'txt' },
-  { type: 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', ext: 'docx' }
+// ENum for supported transcript MIME types
+const TRANSCRIPT_MIME_TYPES = {
+  webvtt: 'text/vtt',
+  srt: 'application/x-subrip',
```

Comment:
I got this mime type from [Wikipedia](https://en.wikipedia.org/wiki/SubRip), but as you said if the mime type is set or read differently by different vendors and computers then it would be safe to set multiple allowed mime types.
I think we can let it be either `application/x-subrip` or `text/srt` safely, but using `text/plain` might be tricky. I'll look into that.

Reply:
Yeah, I don't think we should worry too much about the `text/plain` case. May be worth taking a quick look, but we support parsing .txt files so an SRT coming in with that mime type would lose out on the timed text functionality but the content would still display and be downloadable, I think?

Reply:
Yes, that would be the case.

I changed the parsing code, to first look for the `format` property in the `Annotation` body, instead of only relying on the MIME-type in the `Content-Type` header in the GET request, which would allow more flexibility in identifying the file types in the transcript parser.